tools/debian_config.json

```diff
@@ -5,7 +5,7 @@
     "package_name": "aspnetcore-store",
     "install_root": "/usr/share/dotnet",
 
-    "short_description": "Microsoft ASP.NET Core Runtime Package Store 2.0.0 - Preview 3",
+    "short_description": "Microsoft ASP.NET Core Runtime Package Store 2.0.0",
```

Comment:
It would be good to find a way to pull the version number from version.props. Seems very possible that we'll forget to update this in the future.